homeassistant/components/sensor/forecast.py

```diff
@@ -56,6 +59,16 @@
                              'mm', 'in', 'mm', 'mm', 'mm'],
 }
 DEFAULT_NAME = "Forecast.io"
+CONF_UNITS = 'units'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_CONDITIONS):
+        vol.All(cv.ensure_list, [vol.In(list(SENSOR_TYPES.keys()))]),
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
```

Comment:
You can set a default with `default=` like `vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,`.